IntroToCSharp/Shared/Components/Activities/HorizontalDivider.razor

```diff
@@ -0,0 +1 @@
+<MudDivider Class="mt-6 mb-6" DividerType="DividerType.FullWidth"></MudDivider>
```

Comment:
It feels a little silly to add an entire component with no parameters to do this. Are we sure we need a horizontal divider component?